framework/doc/content/syntax/InterfaceKernels/index.md

```diff
@@ -21,6 +21,18 @@ Interface kernels can be used to provide any general flux condition at an interf
 - `neighbor_var`: The "secondary" variable. This could be $c_1$ from our example above.
 - `boundary`: The interfacial boundary between the subdomains. Note that this must be a sideset and again must exist on the same subdomain as the primary variable. The fact that this boundary is a sideset allows access to variable gradients.
 
+An important private parameter relevant to displaced mesh calculations is
```

Comment:
Hm - just to make sure. There's no case where we would want this to be a public param, correct?

Reply:
This parameter was only introduced to support cohesive zone and its fake neighbors. I think the day when we make fake neighbors a first class capability in libMesh (e.g. they're actually accessible via calls like `neighbor_ptr` and are thus no longer "fake") is the day that we can make this parameter public. As of right now the way we do cohesive zone is kind of an abuse